src/components/Shared/Header/Header.css

```diff
@@ -2,21 +2,85 @@
 	position: fixed;
 	top: 0;
 	width: 100%;
-	height: 80px;
+	height: 69px;
 	display: flex;
 	flex-direction: row;
 	align-items: center;
 	justify-content: space-between;
-	background: linear-gradient(#71d3d1, #67bcba);
+	background: #C1464E;
 	color: white;
 }
 
 .HeaderLeft {
-	padding-left: 80px;
+	padding-left: 10%;
+	flex: 5 1 auto;
+	display: flex;
+	flex-direction: row;
+	justify-content: left;
+	
+}
+
+.HeaderTitle {
+	cursor: default;
+	font-size: 36px;
+	line-height: 42px;
+	font-family: Roboto;
+	font-style: normal;
+	font-weight: normal;;
+}
+
+.HeaderLogo {
+	height: 38px;
+	padding-right: 10px;
+	padding-bottom: 8px;
+
 }
 
 .HeaderRight {
+	flex: 2 1 auto;
 	display: flex;
 	flex-direction: row;
-	padding-right: 80px;
-}
\ No newline at end of file
+	justify-content: right;
+	padding-right: 10%;
+}
+
+.HeaderButton {
+    cursor: pointer;
+    padding-left: 30px;
+    padding-right: 30px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+}
+
+.HeaderChannel {
+	width: 40px;
+	height: 25px;
+	padding-right: 13px;
+}
+
+.HeaderSettings {
+	height: 33px;
+}
+
+.HeaderNotif {
+	height: 33px;
+	padding-right: 13px;
+}
+
+.HeaderProfile {
+	 width:40px;
+	 border-radius: 25px;
```

Comment:
To match the mobile UI, profile pictures should not have a colored border. Same with post images